CHANGELOG.md

```diff
@@ -12,7 +12,7 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 
 ### Changed
-
+*   Adds resetters to user-settings page (#724)
```

Comment:
Would it be clearer to categorize this as "Added" rather than as "Changed"?

Reply:
Looks like the Markdown linter didn't like the 3 rather than 1 space between the bullet and the text.

```
CHANGELOG.md
  15:5  warning  Incorrect list-item indent: remove 2 spaces  list-item-indent  remark-lint
```